packages/react-native-reanimated/Common/cpp/reanimated/NativeAnimations/tests/PendingNativeLayoutStartsTest.cc

```
@@ -0,0 +1,470 @@
+  fixture.boundary->mountedStates[{1, 12}] = {rect(0, 0, 100, 50)};
+  fixture.pendingStarts->enqueue(
+      fixture.makeRecord(restartedHandle, rect(0, 0, 100, 50), restartedResults, pendingOpacityTrack()));
+  fixture.boundary->simulateMount(1);
+  fixture.scheduler->flush();
+  assert(fixture.executor->started.size() == 2);
+  assert(fixture.executor->started.back() == restartedHandle);
+}
+
+} // namespace
+
+void runPendingNativeLayoutStartsTests() {
```

Cppcheck (2.21.0)

[style] The function 'runPendingNativeLayoutStartsTests' is never used.

(unusedFunction)